### PR TITLE
ona integration

### PR DESCRIPTION
**Pull request by rudigiesler**

### CODE REVIEW

**Comment by rudigiesler** [comment]
@hodgestar: Ready for review. We will need to add the proper `require` once the Ona helper is released.

File: test/app.test.js [hunk 1]
```diff
@@ -538,8 +581,10 @@ describe("App", function() {
                 .setup.user.addr('+12345')
                 .inputs('MN34', '1')
                 .check(function(api, im , app) {
-                    http_sent = api.http.requests[api.http.requests.length-1];
-                    assert.equal(http_sent.url, 'http://besnappy.com/api/');
+                    var index = _.findIndex(api.http.requests, {
+                        url: 'http://besnappy.com/api/'
+                    });
+                    http_sent = api.http.requests[index];
```
Comment (hodgestar) [inline test/app.test.js]: Can we maybe do this in one step with `_.where(...)`?

**Comment by hodgestar** [comment]
Otherwise looks good. We should release the Ona helper before landing.

**Comment by hodgestar** [comment]
:+1:

**Pull request merged**